File: support/src/main/java/com/wuguangxin/base/LoadingStatus.java
package com.wuguangxin.base;

/**
 * 加载状态。
 *
 * Created by wuguangxin on 16/10/13
 */
public interface LoadingStatus {
	/**
	 * 请求开始
	 */
	int START = 100;
	
	/**
	 * 文件大小
	 */
	int COUNT = 110;
	
	/**
	 * 下载进度
	 */
	int PROGRESS = 111;
	
	/**
	 * 请求成功
	 */
	int SUCCESS = 200;
	
	/**
	 * 请求取消
	 */
	int CANCEL = 300;
	
	/**
	 * 请求失败
	 */
	int FAILURE = 400;
	
	/**
	 * 请求完成
	 */
	int FINISH = 900;
}

File: support/src/main/java/com/wuguangxin/base/LoadingStatus.kt
package com.wuguangxin.base

/**
 * 加载状态。
 *
 * Created by wuguangxin on 16/10/13
 */
object LoadingStatus {
    /**
     * 请求开始
     */
    const val START = 100

    /**
     * 文件大小
     */
    const val COUNT = 110

    /**
     * 下载进度
     */
    const val PROGRESS = 111

    /**
     * 请求成功
     */
    const val SUCCESS = 200

    /**
     * 请求取消
     */
    const val CANCEL = 300

    /**
     * 请求失败
     */
    const val FAILURE = 400

    /**
     * 请求完成
     */
    const val FINISH = 900
}